components/hab/static/template_gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
```

Comment:
Tangentially related... the name `results` is rather general. I wonder if we should consider renaming to `hab-results`, `hab-artifacts`, etc.; basically make it obvious that this is a Habitat-related directory.